src/sentry/static/sentry/app/components/panels/panelItemGroup.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from 'react-emotion';
+
+import PanelItem from 'app/components/panels/panelItem';
+
+const PanelItemGroup = styled.div`
+  border-left: 4px solid ${p => p.theme.borderLight};
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  background: ${p => p.theme.offWhite};
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  ${PanelItem} {
```

Comment:
Yeah imo, we should style components where the React component is defined, otherwise it's easy to overlook that another component is overriding your styles. 

I think it does feel odd that the child is importing the parent,  but having all styles contained within its component definition module is a nice benefit.

Thoughts on this @Chrissy @ckj ?

Reply:
Might make sense for this to do a relative import actually

Reply:
yeah I agree. this situation is a bit odd since the parent is basically a styled wrapping container, but in general we want to avoid scoping as much as possible and keep styles localized to the markup they affect.